scripts/devops_tasks/find_change_log.ps1

```diff
@@ -15,6 +15,7 @@ function ExtractReleaseNotes($changeLogLocation)
     return $releaseNotes
   }
 
+  Write-Host $changeLogLocation
```

Comment:
Debugging? If not and permanent, probably needs a bit of description.

Reply:
Its a debug line and forgot to remove before pushing files.